tsl/test/expected/bgw_custom.out

```diff
@@ -26,6 +26,19 @@ BEGIN
   COMMIT;
 END
 $$;
+\set ON_ERROR_STOP 0
+-- test bad input
+SELECT add_job(NULL, '1h');
+ERROR:  function cannot be NULL
+SELECT add_job('invalid_func', '1h');
+ERROR:  function "invalid_func" does not exist at character 16
```

Comment:
It can be a procedure too, so it is better to not focus just on function. For example, user call `add_job` for a procedure, but misspelled the name, then the error message can be interpreted as it is required to have a function, not a procedure.

Reply:
This is a PostgreSQL error so there is nothing we can do about it. I guess all errors are legacy (focusing on function) since procedures was a recent addition to PostgreSQL.